cubi_tk/sea_snap/check_irods.py

```diff
@@ -77,7 +77,7 @@ def execute(self):
 
         # --- get lists
         # files on SODAR
-        files = self.get_files()
+        files = self.get_data_objs()
         files_rel = [f.replace(self.args.irods_path + "/", "") for f in files["files"]]
         logger.info("Files on SODAR (first 20): %s", ", ".join(files_rel[:19]))
```

Comment:
Again, can we make 20 configurable?

Reply:
Yes, will do.

Reply:
Oh, this is in the sea-snap version. Haven't touched it, not sure how it should be modified or how to test it since I've never dealt with sea-snap. Hints welcome :)